src/DataLayer/KeyMotorData/I_KeyMotorData.h

```diff
@@ -19,9 +19,13 @@ class I_KeyMotorData : public QObject
 signals:
     void motorZeroReceived(KeyMotor);
     void motorOneReceived(KeyMotor);
-    void motorSetPowerReceived(double);
-    void motorSetCurrentReceived(double);
+    void motorZeroSetPowerReceived(double);
```

Comment:
Instead of adding new signals, we should instead use the existing `motorZero/motorOneReceived` signals, that should clean the amount of signals and slots up a lot.

Reply:
The set power calculations can stay like this though, but we're actually using the bus current/ bus voltage, we should rename it to `motorZeroBusPowerReceived` (This was a mistake in the original code). Same goes for motor one